### DISCUSSION TITLE
Unstable Connections (Reconnect) still a feature?

### DISCUSSION THREAD

**Discussion by SJLennon** [codefori/vscode-ibmi#2040] [General]
@worksofliam 
Found this in the help, but is "`Show Reconnect Button`” still a feature? I have` "code-for-ibmi.showReconnectButton": true,` in my settings.json but I don't see any icon. Maybe it is obsolete and has been overtaken by new connection technology?

![image](https://github.com/codefori/vscode-ibmi/assets/67484051/129bcfd4-49e3-424b-888e-421c7c0db593)

If it is obsolete, I will take it out.

**Comment by sebjulliand** [comment]
This has indeed been removed a few moons ago.
Code for i will now automatically offer to reconnect if the connection is lost.
